### DISCUSSION TITLE
Modal with fixed header & footer and scrollable body in Figma

### DISCUSSION THREAD

**Discussion by wangzhenux** [twilio-labs/paste#2434] [Ideas] [answered]
On the paste site, for the Modal component, it said "the Modal will grow in height with the amount of content in it. Once the modal reaches 90% of the height of the viewport, the body will begin to scroll." 

Maybe we can provide a variant of Modal component with fixed header & footer and scrollable body in the Figma.

**Comment by andioneto** [comment]
Hi @wangzhenux 👋 

I think we are still on a feature freeze, but I need to confirm with the rest of the team. I just shared this request with our design team and will let you know what they say when we have an update.

Thank you!

**Comment by loreina** [comment]
Hi @wangzhenux, 

This is a great idea! If you're looking to use this for prototyping, Modal is affected by slot components work so after that's published, we'll look deeper into this alongside interactive components.

If you're looking to use this for for static mocks, feel free to mess around with auto-layout a bit to create this affect (set Modal to Fixed, set Container to Fill, set Content to Fill and Clip Content).

In the meantime, I've created a ticket in the backlog and the team will follow up when we work on it. Thanks!

**Comment by loreina** [comment, marked as answer]
Marking this as answered due to inactivity. Unfortunately, we likely won't get around to supporting a variant for max width at this time as it's not a native feature in Figma. Thanks!
